instance/models/openedx_instance.py

```diff
@@ -274,9 +274,14 @@ def get_load_balancer_configuration(self):
         """
         if not self.active_appserver:
             return self.get_preliminary_page_config(self.ref.pk)
+        ip_address = self.active_appserver.server.public_ip
+        if not ip_address:
+            # The active appserver doesn't have a public IP address.  This means that the server
+            # has been terminated, so we don't show the preliminary page in this case, and simply
+            # deconfigure the backend instead.
+            return [], []
```

Comment:
@smarnach I meant that we should log an error here, and show that error to the user, not just silently act like if this was a normal situation. If the load balancer is instructed to point to a server which doesn't have an IP, something went wrong somewhere, and the more proactive we are in showing that error, the easier it will be to figure it out.